ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -153,12 +153,18 @@ export const itwEidIssuanceMachine = setup({
             "trackWalletInstanceRevocation"
           ]
         },
-        onError: {
-          actions: assign(
-            setFailure(IssuanceFailureType.WALLET_REVOCATION_GENERIC)
-          ),
-          target: "#itwEidIssuanceMachine.Failure"
-        }
+        onError: [
+          {
+            guard: "isSessionExpired",
+            target: "SessionExpired"
```

Comment:
I think we should adjust the `SessionExpired` state as well. Currently it transitions the machine to `TosAcceptance` (basically restarting it), because the session expired could only happen at the beginning of the issuance flow, but this time we should go back to `WalletInstanceRevocation`.

Reply:
Addressed here: https://github.com/pagopa/io-app/pull/6371/commits/e5a96fd45aed16b8663adec3a15f5baee1182b97
I was able to handle correctly the 401 status code by removing the `SessionExpired` state, which was a placeholder state used to trigger an action before transitioning to a different state.